### ISSUE TITLE
title field reset after change in serie field

### ISSUE THREAD

**Issue by ghost** [eleybourn/Book-Catalogue#636]
I've had this issue for quite some time now, and I thought there was already an issue about it, but can't find it anymore.

Here is the case. I had a book by scanning it's barcode. Many fields are already completed by the search, but quite often, the Serie field is not filled. In fact, quite often, the serie data is found in the title field, along with the title.
I correct the title, and then enter the serie form to add the serie. Either by canceling the action or by validating the serie added, I'm brought back to the main form, and find that the title field has been reset to its search value.
In fact, I have to complete author(s) and serie(s) before touching the title field. Though I haven't tested it, I guess the same goes for any field on the main form. As soon as you go to a sub-form (author or serie, for example), the fields reset to their search value.

I've had this issue since Android 4.1 (as far as I remember) and up to 4.4 (not ye tested on 5.0), sunce BookCatalogue 5.0 up to the latest version (few days ago, which prompted me to check this issue once again). I can reproduce it anytime.

**Comment by buttercookie42** [comment]
> Though I haven't tested it, I guess the same goes for any field on the main form. As soon as you go to a sub-form (author or serie, for example), the fields reset to their search value.

Yes, I've had this happen with other fields on the main form, too.
